HospitalGame_Business/src/business/BusinessHighScore.java

```diff
@@ -10,67 +10,120 @@
 import java.util.HashMap;
 import java.util.Map.Entry;
 
-/**
- *
- * @author andreasmolgaard-andersen
- */
+    /**
+     * Class to handle all highscore related functions
+    *
+    * @author Frederik Schultz Rosenberg
+    * @author Andreas Bøgh Mølgaard-Andersen
+    * @author Lars Bjerregaard Jørgensen
+    * @author Robert Francisti
+    */
 public class BusinessHighScore implements IHighScore {
-    
+
     /**
      * How many high scores is saved/max
      */
     private final int store = 10;
-    
+
     /**
      * If the high scores has been updated
      */
     private boolean isDirty = false;
-    
+
     private HashMap<String, Integer> scores;
 
+    /**
+     * gets highScore
+     *
+     * @return scores
+     */
     @Override
     public HashMap<String, Integer> getHighScore() {
         return scores;
     }
 
+    /**
+     * adds HashMap to scores loads all highscores
+     *
+     * @param highScore the highscore
+     */
+
     public void load(IHighScore highScore) {
         scores = new HashMap<>();
-        if (highScore == null) return;
+        if (highScore == null) {
+            return;
+        }
         scores.putAll(highScore.getHighScore());
     }
 
+    /**
+     * checks if highscore has been changed
+     *
+     * @return true if it has been changed
+     */
     public boolean isDirty() {
         return isDirty;
     }
 
+    /**
+     * checks if the score is eligible for a highscore
+     *
+     * @param score the score that represents a HighScore
+     * @return a place on the highscore
+     */
     public boolean eligibleForHighscore(int score) {
```

Comment:
It does not return a place on the high score. It returns true if the score is high score is enough to get on the high score list